x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/EqlSpec.java

```diff
@@ -9,12 +9,15 @@
 import org.elasticsearch.common.Strings;
 
 import java.util.Arrays;
+import java.util.Objects;
 
 public class EqlSpec {
     private String description;
     private String note;
     private String[] tags;
     private String query;
+    private Boolean caseSensitiveOnly;
+    private Boolean caseInsensitiveOnly;
```

Comment:
switched to `boolean`, thanks @aleksmaus. I handle `null => false` within the SpecLoader.

I'm not sure the ideal solution to avoid reuse while keeping it fairly simple. at the least, I can add a comment like you suggest for both classes

Reply:
will leave this PR as is, and update in #57568